integration-tests/tester/tests/snapshot/snapshot_test.go

```diff
@@ -26,8 +26,8 @@ func TestSnapshot(t *testing.T) {
 			cfg.INXCoo.RunAsCoo = false
 		}
 		// modify to use different snapshot files
-		cfg.Snapshot.FullSnapshotFilePath = "/assets/test_full_snapshot.bin"
-		cfg.Snapshot.DeltaSnapshotFilePath = "/assets/test_delta_snapshot.bin"
+		cfg.Snapshot.FullSnapshotFilePath = "/assets/snapshot_full_snapshot.bin"
```

Comment:
why this?

Reply:
I was confused by all the different assets, and in what tests they are used. So I decided to rename the files depending on the integration test they are used in :walking: